### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by kamil-kaczmarek**
Package with:

* note to install plotly to have interactive visuals
* dropped "and current latest"
* added list of all integrations
* moved downloading from the UI to "organizing and exploring exp results" as new page
* added video how to download image in full size
* bumped PL version in integrations.

### CODE REVIEW

File: docs/integrations/index.rst [hunk 1]
```diff
@@ -15,20 +15,45 @@ Integrations are written using |neptune-client| and provide a convenient way to
...
+* :ref:`Pytorch Ignite <integrations-pytorch-ignite>`
+* :ref:`Pytorch Lightning <integrations-pytorch-lightning>`
+* :ref:`Sacred <integrations-sacred>`
+* :ref:`Scikit Optimize <integrations-scikit-optimize>`
+* :ref:`Skorch <integrations-skorch>`
+* :ref:`Telegram <integrations-telegram>`
+* :ref:`Tensorboard <integrations-tensorboard>`
+* :ref:`Tensorflow <integrations-tensorflow>`
+* :ref:`XGBoost <integrations-xgboost>`
+* :ref:`Scikit-learn <list-of-all-integrations>` *coming soon...*
```
Comment (jakubczakon) [inline docs/integrations/index.rst:51]: I love this!

File: docs/integrations/pytorch_lightning.rst [hunk 2]
```diff
@@ -22,7 +22,7 @@ PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI resea
 
 .. note::
 
-    This integration is tested with ``pytorch-lightning==1.0.0`` and current latest, and ``neptune-client==0.4.123`` and current latest.
+    This integration is tested with ``pytorch-lightning==1.0.7``, and ``neptune-client==0.4.123``.
```
Comment (jakubczakon) [inline docs/integrations/pytorch_lightning.rst:25]: I think (if the examples didn't fail) it works with neptune-client==0.4.126

File: docs/integrations/pytorch_lightning.rst [hunk 3]
```diff
@@ -53,7 +53,7 @@ You have ``Python 3.x`` and following libraries installed:
 
 * ``neptune-client==0.4.123`` or newer: See :ref:`neptune-client installation guide <installation-neptune-client>`.
```
Comment (jakubczakon) [inline docs/integrations/pytorch_lightning.rst:54]: I think (if the examples didn't fail) it works with neptune-client==0.4.126

**Review by jakubczakon — CHANGES_REQUESTED** [review]

File: docs/integrations/index.rst (hunk 1 above)
Comment (kamil-kaczmarek) [reply docs/integrations/index.rst:51]: thanks!

File: docs/integrations/pytorch_lightning.rst (hunk 2 above)
Comment (kamil-kaczmarek) [reply docs/integrations/pytorch_lightning.rst:25]: yep, corrected.

File: docs/integrations/pytorch_lightning.rst (hunk 3 above)
Comment (kamil-kaczmarek) [reply docs/integrations/pytorch_lightning.rst:54]: fixed

**Review by jakubczakon — APPROVED** [review]

**Pull request merged**